webanno-api-dao/src/test/java/de/tudarmstadt/ukp/clarin/webanno/api/dao/CasStorageServiceImplTest.java

```diff
@@ -93,6 +93,9 @@ public void testWriteReadExistsDeleteCas() throws Exception
         sut.deleteCas(doc, user);
         assertThat(sut.getCasFile(doc, user)).doesNotExist();
         assertThat(sut.existsCas(doc, user)).isFalse();
+        
+        // check that cas is no longer in the active session
+        assertThat(casStorageSession.contains(cas.getCas())).isFalse();
```

Comment:
@reckart this test is currently failing because we have neither set a casholder in the session or in the withexclusiveacess. do you know if cases like this exist in real life?

Reply:
This test was asserting against the wrong CAS - against the original template CAS, not against the CAS that was read (into the session) and then deleted.